<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>781</width>
      <height>551</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>2</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>Tab 1</string>
     </attribute>
     <widget class="QPushButton" name="btn_menuPersonal">
      <property name="geometry">
       <rect>
        <x>300</x>
        <y>160</y>
        <width>161</width>
        <height>61</height>
       </rect>
      </property>
      <property name="text">
       <string>Menú Empleado</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btn_menuCliente">
      <property name="geometry">
       <rect>
        <x>300</x>
        <y>240</y>
        <width>161</width>
        <height>61</height>
       </rect>
      </property>
      <property name="text">
       <string>Menú Cliente</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btn_salir">
      <property name="geometry">
       <rect>
        <x>300</x>
        <y>320</y>
        <width>161</width>
        <height>61</height>
       </rect>
      </property>
      <property name="text">
       <string>Salir</string>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>60</y>
        <width>551</width>
        <height>71</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Sitka Display Semibold</family>
        <pointsize>48</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>MENÚ</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignmentFlag::AlignCenter</set>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>Tab 2</string>
     </attribute>
    </widget>
    <widget class="QWidget" name="tab_3">
     <attribute name="title">
      <string>Page</string>
     </attribute>
     <widget class="QPushButton" name="btn_reservar">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>190</y>
        <width>191</width>
        <height>81</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Sitka Display</family>
        <pointsize>11</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Reservar Mesa</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btn_cancelarReserva">
      <property name="geometry">
       <rect>
        <x>400</x>
        <y>300</y>
        <width>191</width>
        <height>81</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Sitka Display</family>
        <pointsize>11</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Cancelar Reserva</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btn_salir_4">
      <property name="geometry">
       <rect>
        <x>400</x>
        <y>190</y>
        <width>191</width>
        <height>81</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Sitka Display</family>
        <pointsize>11</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Modificar Reserva</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btn_salir_5">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>300</y>
        <width>191</width>
        <height>81</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Sitka Display</family>
        <pointsize>11</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Consultar 
disponibilidad 
de mesas</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>60</y>
        <width>581</width>
        <height>101</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Sitka Display Semibold</family>
        <pointsize>48</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>MENÚ CLIENTE</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignmentFlag::AlignCenter</set>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_4">
     <attribute name="title">
      <string>Page</string>
     </attribute>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>150</y>
        <width>63</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Nombre</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>30</y>
        <width>581</width>
        <height>101</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Sitka Display Semibold</family>
        <pointsize>36</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Reserva de Mesas</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignmentFlag::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>230</y>
        <width>141</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Número telefónico</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>470</x>
        <y>150</y>
        <width>171</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Número de comensales:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_7">
      <property name="geometry">
       <rect>
        <x>470</x>
        <y>230</y>
        <width>61</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Fecha</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_8">
      <property name="geometry">
       <rect>
        <x>470</x>
        <y>310</y>
        <width>61</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Hora</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lE_nombreReserva">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>180</y>
        <width>241</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="lE_numTel">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>260</y>
        <width>241</width>
        <height>28</height>
       </rect>
      </property>
     </widget>
     <widget class="QSpinBox" name="spinBox_numComensales">
      <property name="geometry">
       <rect>
        <x>470</x>
        <y>180</y>
        <width>161</width>
        <height>41</height>
       </rect>
      </property>
     </widget>
     <widget class="QTimeEdit" name="timeEdit_horaReserva">
      <property name="geometry">
       <rect>
        <x>470</x>
        <y>340</y>
        <width>221</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QDateEdit" name="dateEdit_fechaReserva">
      <property name="geometry">
       <rect>
        <x>470</x>
        <y>260</y>
        <width>221</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="btn_aceptarReserva">
      <property name="geometry">
       <rect>
        <x>310</x>
        <y>410</y>
        <width>151</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string>Reservar</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_9">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>310</y>
        <width>161</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Número de identidad</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lE_id">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>340</y>
        <width>241</width>
        <height>28</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_10">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>110</y>
        <width>291</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Ingrese sus datos para realizar una reserva</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignmentFlag::AlignCenter</set>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_6">
     <attribute name="title">
      <string>Page</string>
     </attribute>
     <widget class="QLabel" name="label_11">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>20</y>
        <width>581</width>
        <height>101</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Sitka Display Semibold</family>
        <pointsize>36</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Cancelar Reserva</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignmentFlag::AlignCenter</set>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_5">
     <attribute name="title">
      <string>Page</string>
     </attribute>
     <widget class="QPushButton" name="btn_numMesas">
      <property name="geometry">
       <rect>
        <x>400</x>
        <y>220</y>
        <width>151</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string>Reservar</string>
      </property>
     </widget>
     <widget class="QSpinBox" name="spinBox_numMesas">
      <property name="geometry">
       <rect>
        <x>220</x>
        <y>100</y>
        <width>161</width>
        <height>51</height>
       </rect>
      </property>
     </widget>
     <widget class="QTextEdit" name="tE_Reservas">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>180</y>
        <width>291</width>
        <height>251</height>
       </rect>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
